sdk/test/NetStandard/IntegrationTests/IntegrationTests/CredentialsTests.cs

```diff
@@ -47,5 +50,38 @@ public async Task TestSessionCredentials()
                 }
             }
         }
+
+        [Fact]
+        [Trait(TestBase.CategoryAttribute, "Credentials")]
+        public async Task TestAssumeRoleAWSCredentialsAsyncGet()
+        {
+            using (var sts = TestBase.CreateClient<AmazonSecurityTokenServiceClient>())
+            using (var iam = TestBase.CreateClient<AmazonIdentityManagementServiceClient>())
+            {
+                var callerIdentity = await sts.GetCallerIdentityAsync(new GetCallerIdentityRequest());
+
+                var roleName = "sdk-testrole-" + DateTime.UtcNow.Ticks;
+                var createRoleRequest = new CreateRoleRequest()
+                {
+                    RoleName = roleName,
+                    AssumeRolePolicyDocument = "{\"Version\": \"2012-10-17\", \"Statement\": [{ \"Effect\": \"Allow\",\"Principal\": {\"AWS\": [\"" + callerIdentity.Account + "\"]},\"Action\": \"sts:AssumeRole\",\"Condition\": {}}]}"
+                };
+                var role = await iam.CreateRoleAsync(createRoleRequest);
+
+                var sourceCredentials = FallbackCredentialsFactory.GetCredentials();
```

Comment:
The `FallbackCredentialsFactory` is deprecated and it should not be used anymore (the replacement is `DefaultAWSCredentialsIdentityResolver` but as I mentioned in the other comment this test doesn't run in the build system so in my opinion we should remove it).

Reply:
Removed test case in latest commit https://github.com/aws/aws-sdk-net/pull/3681/commits/e9ede6c25ed4a3e7e2ff9ea18b0ce7ccefcf7750. Will initiate a dry run.